src/screen/GameScreen.java

```diff
@@ -771,10 +768,21 @@ && checkCollision(bullet, this.enemyShipSpecial)) {
 			}
 		}
 		for (Item item : this.items){
-			if(checkCollision(item, this.ship) && !this.levelFinished){
-				recyclableItem.add(item);
-				this.logger.info("Get Item ");
-
+			if (checkCollision(item, this.ship) && !this.levelFinished) {
+				/* version 1.0 pause effect when player eat item. */
+					/*if(checkCollision(item, this.ship) && !this.levelFinished){
+						recyclableItem.add(item);
+						this.logger.info("Get Item ");
+						Thread thread = new Thread();
+						thread.start();
+						try{
+							for(int i=0;i<3;i++){
+								Thread.sleep(300);
+							}
+						} catch(Exception e){
+							e.printStackTrace();
+						}
+						 */
```

Comment:
[P3]

There is no fault. In particular, it is good to leave the old code while processing comments out in this part.